packages/accounts-base/accounts_common.js

```diff
@@ -170,6 +170,18 @@ export class AccountsCommon {
       : null;
   }
 
+  /**
+   * @summary Get the current user record, or `null` if no user is logged in.
+   * @locus Anywhere
+   * @param {Object} [options]
+   * @param {MongoFieldSpecifier} options.fields Dictionary of fields to return or exclude.
+   */
+  async userAsync(options) {
```

Comment:
Maybe it's a personal preference, but the less `async` functions the better. Wrapping `null` in `Promise.resolve` would have the same effect and generate less code when downleveled.

Reply:
Yeah.
This is more of a preference; having the async keyword in the function signature seems easier to know that it will return a Promise<Some> doing this at the end of the function (wrapping with Promise.resolve) in this case would not be a problem for legibility.
If you think this is a point that we need to discuss further to make it consistent around the codebase, I will be really happy to do it

Reply:
Sure, got that. I just still live in the word of extremely expensive `async`/`await` downleveling.